regression/contracts/invar_check_break_pass/test.desc

```diff
@@ -0,0 +1,8 @@
+CORE
+main.c
+--enforce-all-contracts
+^EXIT=0$
+^SIGNAL=0$
+^VERIFICATION SUCCESSFUL$
```

Comment:
Is there a reason why you don't check the individual VCs as well as the over-all status?